src/Features/Blockcore.Features.ColdStaking/UI/Pages/Modal/ModalViewTransaction.razor

```diff
@@ -0,0 +1,37 @@
+@inject ModalService ModalService
+
+@if (ShowForm)
+{
+
+<button class="btn btn-primary" @onclick="Submit">Submit</button>
+
+} else {
+<div class="row ">
+    <div class="col-12">
+        <div class="card">
+            <div class="card-body">
+                <h5 class="text-success">@Alert</h5>
+                <button class="btn btn-primary" @onclick="CloseModal">Close</button>
+            </div>
+        </div>
+    </div>
+</div>
+}
+
+@code
+{
+	bool ShowForm { get; set; } = true;
+    string Alert { get; set; }
+ 
+    private void CloseModal()
+    {
+        ModalService.Close();
+        //NavigationManager.NavigateTo("walletview/" + WalletName + "/account 0");
+    }
+
+	private void Submit()
+	{
+		ShowForm = false;
+		this.Alert = "Your wallet has now been created.";
```

Comment:
Is this alert correct?

Reply:
No the view tx is not implemented yet, I still need to work out how I’m going to do it. I may disable this until I can get it working so we can push the other changes

Reply:
> > Display segwit address when segwit enabled
> 
> I see that you took away the non segwit address textbox (I wonder if we should keep it just in case?
> Are there situations a none segwit address is needed even if segwit is enabled?

We can add this back in if necessary, I'll leave it as is for now.